akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -416,17 +417,22 @@ private[http] object HttpServerBluePrint {
             log.warning(
               """Sending 2xx response before end of request was received...
                 |Note that the connection will be closed after this response. Also, many clients will not read early responses!
-                |Consider waiting for the request end before dispatching this response!""".stripMargin)
+                |Consider only issuing this response after the request data has been completely read!""".stripMargin)
```

Comment:
While I was here wanted to cleanup the early response warning to be more clear what we want from people here